### DISCUSSION TITLE
Building & serving GitBook locally?

### DISCUSSION THREAD

**Discussion by kyduff** [GitbookIO/community#119] [0. General bugs and questions] [answered]
### Ask a question, start a conversation, or make an annoucement

I'm trying to build + serve a gitbook locally on my machine (MacOS Monterey 12.6.3). I can see that [`gitbook-cli`](https://github.com/GitbookIO/gitbook-cli) is no longer supported and now breaks on my system. Is there another way to just build a gitbook locally without needing to go through the app / GitHub integrations?

**Comment by aikoii** [comment, marked as answer]
Hi @kyduff!

As the efforts of the GitBook team are focused on the [GitBook.com](https://www.gitbook.com/) platform, as you already noticed, the CLI is no longer under active development. This is why you're running into issues.

I'd recommend using the current version of GitBook for your documentation. 🙂

**Comment by JohnAtQNX** [comment]
Hiya! Just want to share that the mdBook utility (maintained by Rust) seems to be able to build the GitBook sources without any modification, as long as you have Git Sync available to extract your sources. Of course it won't look the same as GitBook.com, but it can serve as an offline render. Some things benefit from some tweaks (like changing all headers to `#` from `##` in summary.md to support TOC groups), but it should build in place without trouble.

**Comment by addisonschultz** [comment]
Hey @kyduff, as Aiko mentioned, the CLI is unfortunately not supported anymore in the latest version of GitBook. I'd love to know though, what your use-cases are to use the CLI over the newest version of GitBook? We're interested in making sure the latest version has all the features and workflows supported that were previously there, so if there's something specific you're missing, I'd love to hear!
